File: VetAPI/Models/Tutor.cs
namespace VetAPI.Models
{
    public class Tutor
    {
        public int TutorId { get; set;}
        public string? Nome { get; set;}
        public string? Cpf { get; set;}
        public DateTime? DataNascimento { get; set;}
        public string? Telefone {get;set;}
        public string? Email{get;set;}
        public List<Animal>? Animais { get; set; } = new List<Animal>();
    }
}

File: VetAPP/src/app/Tutor.ts
import { Animal } from './Animal';

export class Tutor {
    tutorId: number;
    nomeTutor: string = '';
    cpf: string = '';
    dataNascimento: Date;
    telefone: string = '';
    email: string = '';
    animais: Animal[];

    constructor(tutorId: number, dataNascimento: Date) {
        this.tutorId = tutorId;
        this.dataNascimento = dataNascimento;
        this.animais = [];
    }
}